content/en/docs/concepts/components.md

```diff
@@ -88,15 +88,18 @@ For more information, see
 
 ### Resource Detectors
 
-A resource represents the entity producing telemetry as resource attributes. For
-example, a process producing telemetry that is running in a container on
-Kubernetes has a Pod name, a namespace, and possibly a deployment name. All
-three of these attributes can be included in the resource.
+A [resource](/docs/concepts/resources/) represents the entity producing
```

Comment:
hrm. This page doesn't exist?
<img width="503" alt="image" src="https://github.com/open-telemetry/opentelemetry.io/assets/6309070/e193a9a7-e495-4a62-9bf6-afd4a78f14bf">

Reply:
oh... it looks like I missed to add that page to my PR, odd...

Reply:
```
$ git add content/en/docs/concepts/resources/_index.md
The following paths are ignored by one of your .gitignore files:
content/en/docs/concepts/resources
hint: Use -f if you really want to add them.
hint: Turn this message off by running
hint: "git config advice.addIgnoredFile false"
$ head .gitignore
# Hugo-generated assets
.hugo_build.lock
/public
resources/
```

Reply:
oh shit, well that's fine. Does it really need to be a subfolder though? I think just a markdown file for now is good enough

Reply:
There's an image, that's why I need a folder, I updated the `.gitignore` to only apply to top-level resources/ folder, hope that's not going to be a problem in the future

Reply:
fixed, PTAL